content/docs/1.5.0/references/networking.md

```diff
@@ -45,16 +46,16 @@ To | Port | Protocol
 #### ingress
 From | Port | Protocol
 --- | --- | ---
-`Longhorn Manager` | 8500; 10000-30000 | TCP
+`Longhorn Manager` | 8500; 8501 | TCP
 `Other Instance Manager` | 10000-30000 | TCP
```

Comment:
I found the syncAgent in longhorn-engine would listen on `9504`
and use `9700-9800` for syncing files between replicas
should also be added to `other Instance Manager`
https://github.com/longhorn/longhorn-engine/blob/master/app/cmd/sync_agent.go#L21-L33

Reply:
Currently, this cmd is not used by anyone. The sync agent server will be launched along with replica servers, and replica servers use port 10000-30000